app/scripts/controllers/HomeCtrl.js

```diff
@@ -1,16 +1,15 @@
 (function() {
-  function HomeCtrl(ApiRequests, $cookies) {
-      this.ApiRequests = ApiRequests;
+  function HomeCtrl(ListsApiRequests, $cookies, UsersApiRequests) {
```

Comment:
I'd organize this with your custom services both last and the Angular object (`$cookies`) first.